packages/help-center/src/hooks/use-get-history-chats.ts

```diff
@@ -58,7 +58,24 @@ export const useGetHistoryChats = (): UseGetHistoryChatsResult => {
 				allConversations,
 				allSupportInteractions
 			);
-			setConversations( filteredConversations );
+
+			const conversationsWithUpdatedStatuses = filteredConversations.map( ( conversation ) => {
+				const supportInteraction = allSupportInteractions.find(
+					( interaction ) => interaction.uuid === conversation.metadata?.supportInteractionId
+				);
+
+				const updatedConversation = {
+					...conversation,
+					metadata: {
```

Comment:
Hmm, is it possible we don't have `conversation.metadata`? I see we apply some optional chaining to it when finding the related support interaction above.

Reply:
The `metadata` property will always exist. It looks like it's always added when a new conversation is created.

Reply:
I understand. But if the `metadata` property will exist, can we reflect that in the callback above? Right now we still have optional chaining: `conversation.metadata?.supportInteractionId`.